CONFIGDOC.md

```diff
@@ -97,6 +108,201 @@ You can define several axes in the **Configuration Matrix** section. For example
 
 * A multi-configuration project creates a separate workspace for each user-defined axis value. If you specify the full paths to where MATLAB is installed as axis values, Jenkins fails to create separate workspaces and fails the build.
 
+## Set Up Pipeline Project
+When you define your Pipeline with a `Jenkinsfile`, the plugin provides you with a step to run MATLAB scripts, functions, and statements. The plugin also provides a step to run MATLAB and Simulink tests. (These steps are common to both Declarative and Scripted Pipeline.) 
+
+To configure the plugin for a Pipeline project:
+1) Define your Pipeline in a `Jenkinsfile` in the root of your repository.
+2) In the **Pipeline** section of the project configuration window, select **Pipeline script from SCM** from the **Definition** drop-down menu. 
+3) Select your source control system from the **SCM** drop-down menu.
+4) Paste your repository URL into the **Repository URL** box.
+
+You also can define your Pipeline directly in the project configuration window. If you select **Pipeline script** from the **Definition** drop-down menu, you can author your Pipeline code in the **Script** box. When you define your Pipeline this way, your Pipeline must include an additional stage to check out MATLAB code from source control.

```

Comment:
I am not sure if I understand your comment correctly. Is your comment on this paragraph? Is so, could you please provide your rearranged version? I don't see how the sentences can be rearranged here.

"You also can define your Pipeline directly in the project configuration window. If you select Pipeline script from the Definition drop-down menu, you can author your Pipeline code in the Script box. When you define your Pipeline this way, your Pipeline must include an additional stage to check out MATLAB code from source control."

Reply:
This would be the rearrangement I am suggesting:

When you define your Pipeline this way, your Pipeline must include an additional stage to check out MATLAB code from source control. You also can define your Pipeline directly in the project configuration window. If you select **Pipeline script** from the **Definition** drop-down menu, you can author your Pipeline code in the **Script** box

Reply:
Reading it again, this makes sense Houman. Looks good, keep it the same way

Reply:
Sounds good :).